src/runtime/runtime_nintendoswitch_svc.go

```diff
@@ -0,0 +1,17 @@
+// +build nintendoswitch
```

Comment:
It seems to me that this could be in the runtime_nintendoswitch.go file, or is there a reason to keep it separate?